components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/api/APIGateway.java

```diff
@@ -92,4 +92,13 @@
      */
     void deleteEndpoint(Endpoint endpoint) throws GatewayException;
 
+    /**
+     * Add API state change to gateway
```

Comment:
para meter desccriptions needed and better to change the description to more meaningful manner